go.mod

```diff
@@ -4,6 +4,7 @@ go 1.18
 
 require (
 	github.com/eclipse/paho.mqtt.golang v1.4.1
+	github.com/edgexfoundry/go-mod-core-contracts/v2 v2.3.0-dev.15
```

Comment:
Don't use this just for the single constant. We try not to have this module dependent on any thing else in EdgeX